packages/jest-runtime/src/cli/index.ts

```diff
@@ -81,8 +81,11 @@ export function run(cliArgv?: Config.Argv, cliInfo?: Array<string>) {
     maxWorkers: Math.max(os.cpus().length - 1, 1),
     watchman: globalConfig.watchman,
   }) as Promise<Context>)
-    .then(hasteMap => {
-      const Environment: typeof JestEnvironment = require(config.testEnvironment);
+    .then(async hasteMap => {
+      const transformer = new Runtime.ScriptTransformer(config);
```

Comment:
Can we import `ScriptTransformer` directly rather than go through the static on `Runtime`? Maybe do the `interopDefaultRequire` dance as well

Reply:
Done